### PR TITLE
docs: Added walktrough for Programming Gateway using MPLAB IPE

### PR DESCRIPTION
**Pull request by KrishnaIyer**

### CODE REVIEW

File: _content/gateways/gateway/programhexfile.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+#### 3. Connection
+Click the `connect` button next to the Tool to allow the IPE to connect to the Programmer. This allows the IPE to read existing firmware information.
+
+   **Connection**
+    ![Connection](ipe_2_connected.png)
+
+#### 4. Load Hex File
+Now load the hex file from your PC. The IPE checks it for validity.
+
+   **Loac Hex File**
```
Comment (htdvisser) [inline _content/gateways/gateway/programhexfile.md]: typo

File: _content/gateways/gateway/programhexfile.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    ![Selection](ipe_1_selected.png)
+
+#### 3. Connection
+Click the `connect` button next to the Tool to allow the IPE to connect to the Programmer. This allows the IPE to read existing firmware information.
+
+   **Connection**
+    ![Connection](ipe_2_connected.png)
+
+#### 4. Load Hex File
+Now load the hex file from your PC. The IPE checks it for validity.
```
Comment (htdvisser) [inline _content/gateways/gateway/programhexfile.md:29]: This does not say anything about the IPE warning about addresses that don't exist on the PIC

**Review by htdvisser — CHANGES_REQUESTED** [review]

File: _content/gateways/gateway/programhexfile.md (hunk 1 above)
Comment (KrishnaIyer) [reply _content/gateways/gateway/programhexfile.md]: done

File: _content/gateways/gateway/programhexfile.md (hunk 2 above)
Comment (KrishnaIyer) [reply _content/gateways/gateway/programhexfile.md:29]: @htdvisser: You know why it's happening here?
Usually this issue would be something to do with the `.map` file or the linker scripts

Comment (htdvisser) [reply _content/gateways/gateway/programhexfile.md:29]: I have no idea. I just took the screenshots (as a noob in progging) and noticed the warning

Comment (KrishnaIyer) [reply _content/gateways/gateway/programhexfile.md:29]: Well I programmed another gateway and I didn't see this error. It should be an MPLAB specific thing. I think we can add a comment on this and close this PR

File: _content/gateways/gateway/programhexfile.md (hunk 1 above)
Comment (htdvisser) [reply _content/gateways/gateway/programhexfile.md]: Github still shows the typo. Forgot to push maybe?

Comment (KrishnaIyer) [reply _content/gateways/gateway/programhexfile.md]: oopsey

**Review by htdvisser — APPROVED** [review]

**Pull request merged**